### PR TITLE
Better Dreams

### PR DESCRIPTION
**Pull request by Affurit**
Not fully finished due to ban leading to lack of investment in the server.

Improves dreaming/makes it less out of place for the server's lore.

### CODE REVIEW

**Comment by PrPleGoo** [comment]
I think inconsistent quotes are causing the CI build to fail. The first dream is opened by two quotes. 
`""Welcome to The NSS Vignette!""` This should be giving you a compile error and the color coding from git implies keyword use.

Replace all double (and triple) occurances of quotes with single quotes and press CTRL+K before committing.

File: code/modules/flufftext/Dreaming.dm [hunk 1]
```diff
@@ -1,16 +1,22 @@
...
-	"an ID card","a bottle","a familiar face","a crewmember","a toolbox","a security officer","the captain",
-	"voices from all around","deep space","a doctor","the engine","a traitor","an ally","darkness",
-	"light","a scientist","a monkey","a catastrophe","a loved one","a gun","warmth","freezing","the sun",
-	"a hat","a ruined station","a planet","phoron","air","the medical bay","the bridge","blinking lights",
-	"a blue light","an abandoned laboratory","Nanotrasen", "pirates", "mercenaries","blood","healing","power","respect",
-	"riches","space","a crash","happiness","pride","a fall","water","flames","ice","melons","flying","the eggs","money",
-	"the chief engineer","the research director","the chief medical officer",
-	"a station engineer","the janitor","the atmospheric technician",
-	"a cargo technician","the botanist","a shaft miner","the psychologist","the chemist",
+	""Welcome to The NSS Vignette!"",""A whole new frontier..."",""Life is transient, but so is death."",
```
Comment (Zero-Bits) [inline code/modules/flufftext/Dreaming.dm]: Double quotes mess up the string definitions. Either add a backslash before each instance of the inner quotes (e.g. `"\"Welcome to the NSS Vignette!\""`) or remove them.

File: code/modules/flufftext/Dreaming.dm [hunk 2]
```diff
@@ -1,16 +1,22 @@
...
+	"The gateway annihilates your body, spreading it across the cosmos...","The stars chime like struck bells.",
+	"You hear a distant bark...","A faraway sun smiles at you.","Something whispers into your ear...","Xentec...?","Your lungs!",
+	"You hear a sinister hiss...","The computer beeps, and then goes black.",
+	"A great light from the skies eviscerates an entire colony...",The Darkness...",""Love..."","""Close your eyes."",
```
Comment (Zero-Bits) [inline code/modules/flufftext/Dreaming.dm]: More double quotes and a set of triple quotes. Oof.

File: code/modules/flufftext/Dreaming.dm [hunk 3]
```diff
@@ -1,16 +1,22 @@
...
+	"Lush forests...",""Want another drink?"","The airlock explodes!","The slime envelops your head...",""WAKE UP!"","Snow falls...",
```
Comment (Zero-Bits) [inline code/modules/flufftext/Dreaming.dm]: More double quotes.

File: code/modules/flufftext/Dreaming.dm [hunk 4]
```diff
@@ -1,16 +1,22 @@
...
+	"The shrine glows with an eerie light...",""Your employment on this station has come to an end."","Blood stains...","Find the truth.",
```
Comment (Zero-Bits) [inline code/modules/flufftext/Dreaming.dm]: Additional double quotes.

File: code/modules/flufftext/Dreaming.dm [hunk 5]
```diff
@@ -1,16 +1,22 @@
...
+	"You sign your name on the dotted line.","Exotic scents...","The tastes of another planet...","Luxurious silks...",
+	""Hyperion, Divinity in innovation!"",""Bluespace is perfectly safe!"",""Green planet? We'll just take another!"",
```
Comment (Zero-Bits) [inline code/modules/flufftext/Dreaming.dm]: Ditto.

File: code/modules/flufftext/Dreaming.dm [hunk 6]
```diff
@@ -1,16 +1,22 @@
...
+	""What's a Vox?"","An explosion rocks the station!","You reach a hand up to feel your exposed brain...",""I'm not scared!",
```
Comment (Zero-Bits) [inline code/modules/flufftext/Dreaming.dm]: Ditto.

File: code/modules/flufftext/Dreaming.dm [hunk 7]
```diff
@@ -1,16 +1,22 @@
...
+	"The air alarm turns red...","A claw slashes your face!","A shooting star passes by...", "You feel trapped.",
+	"A feeling of impending doom rests heavily on you...",""You can do it!"","The thing stares at you, an eerie smile on it's 'face'.",
```
Comment (Zero-Bits) [inline code/modules/flufftext/Dreaming.dm]: Ditto.

File: code/modules/flufftext/Dreaming.dm [hunk 8]
```diff
@@ -1,16 +1,22 @@
...
+	"You wake up...","You fall asleep...","You take one more out of the packet...","You feel at peace with the universe.",
+	""I did not have sexual relations with that Unathi!"","Something is missing...","This journey will never end.","Sleep...",
```
Comment (Zero-Bits) [inline code/modules/flufftext/Dreaming.dm]: Ditto.

File: code/modules/flufftext/Dreaming.dm [hunk 9]
```diff
@@ -1,16 +1,22 @@
...
+	"Zzz...","It's so dark...","...","BOOM!","ZAP!","Sss!","Whoosh!",""No!"",""This was a mistake..."","You hear hooves.","It's cold...",
```
Comment (Zero-Bits) [inline code/modules/flufftext/Dreaming.dm]: Ditto.

**Comment by FaurnsGoldberg** [comment]
im just gonna close this, author is banned from serv?

**Pull request closed without merging**